autoscheduler/frontend/src/components/SchedulingPage/SchedulePreview/SchedulePreview.tsx

```diff
@@ -7,11 +7,13 @@ import * as styles from './SchedulePreview.css';
 import ScheduleListItem from './ScheduleListItem/ScheduleListItem';
 import Schedule from '../../../types/Schedule';
 
+export const noSchedulesText = "Click Generate Schedules to find schedules with the classes you've added.";
```

Comment:
```suggestion
export const noSchedulesText = "Click Generate Schedules to find schedules with the courses you've added.";
```

B/c we say `Add course` for the add course card button - per you

Reply:
yeah fair enough, I was looking for feedback on this but I like it more too